### ISSUE TITLE
Update for GEOTRACES 2021 IDP

### ISSUE THREAD

**Issue by profseth** [briochemc/GEOTRACES.jl#6]
The current code is not compatible with the recently-released 2021 GEOTRACES International Data Product. I might be able to fool the current code into reading the 2021 data by renaming the files, but for other users an update would be preferred.

**Comment by briochemc** [comment]
I tried to fix it quickly but a good fix will take a bit of time because in all their wisdom, GEOTRACES folks decided it was a good idea to change the variable names... E.g., Cadmium used to be `var70` in IDP17 and is now `var85` in IDP21... 

Also, the resistance from GEOTRACES to put the data in a public data repository and make it accessible with a direct URL ***really*** bothers me... In that regard, last week's seminar for the IDP21 launch was a disappointment, to say the least. The disappointment only worsened further when I was told that the data committee would "have to discuss it" when I discussed that exact issue around a year ago... 

In short, with less time available for Julia stuff and low motivation (thanks GEOTRACES!), work on this is going to be slow! 😅

**Comment by briochemc** [comment]
Fixed it anyway 👿
